BSseq_pipeline.py

```diff
@@ -153,6 +153,9 @@
 from itertools import chain
 OUTPUT_FILES = list(chain.from_iterable(chain.from_iterable([targets[name]['files'] for name in selected_targets])))
 
+
+# print(OUTPUT_FILES) # when uncommented, this can be useful for debugging.
+
```

Comment:
This seems a bit too obvious to leave it here.

Reply:
ok, we can remove it if you find it unsightly (it only takes a second to add it in again).